src/Auto/Auto.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Auto.cpp
+
+ *
+ *  Created on: Jan 28, 2017
+ *      Author: peter
+ */
+#include "Timer.h"
+#include "Auto.h"
+#include <iostream>
+#include <stdio.h>
+using namespace std;
+
+Auto::Auto(DriveController* kDrive, RobotModel* kShooter) {
```

Comment:
(next PR) We want to add a function to `Auto` to be called from `MainProgram`. If `Teleop.init()` is called, we want to call a function that will call `Done` for all actions.